src/protocols/vnc/vnc.c

```diff
@@ -385,6 +385,17 @@ void* guac_vnc_client_thread(void* data) {
     }
 #endif
 
+    /* Disable remote console (Server input) */
+    if (settings->disable_server_input) {
+        rfbSetServerInputMsg msg;
+        msg.type = rfbSetServerInput;
+        msg.status = 1;
+        msg.pad = 0;
+
+        rfbBool success = WriteToRFBServer(rfb_client, (char*)&msg, sz_rfbSetServerInputMsg);
+        guac_client_log(client, GUAC_LOG_DEBUG, "%s send request to disable server input", success ? "Successfully" : "Failed to");
```

Comment:
This reads well in the case of failure ("Failed to send request ..."), but the grammar gets wonky if things are sent successfully ("Successfully send request ...").

Perhaps it would be better to simply have two separate `guac_client_log()` calls? That would also allow for the failure case to be logged at a higher log level like `GUAC_LOG_WARNING` without getting too cluttered.